ci/actions-templates/linux-builds-template.yaml

```diff
@@ -169,6 +169,14 @@ jobs: # skip-master skip-pr skip-stable
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_DEFAULT_REGION: us-west-1
+      - name: Deploy build to rustup-builds bucket for release team
+        # if: github.event_name == 'push' && github.ref == 'refs/heads/master' && matrix.mode == 'release'
```

Comment:
Do we want to leave these comments around?

Reply:
This was in a work-in-progress. The final version didn't include them anymore.